OpenMage/magento-lts
# Fix iframe removal in TinyMCE

### Description (*)
The TinyMCE WYSIWYG editor removes iframes when the editor is toggled.

### Related Pull Requests
n/a

### Fixed Issues (if relevant)
n/a

### Manual testing scenarios (*)
1. Create a new CMS page.
2. Disable the WYSIWYG editor.
3. Insert `<iframe width="560" height="315" src="https://www.youtube.com/embed/JvBWJ6Lm9MU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>` in the content of the CMS page.
4. Enable the WYSIWYG editor.
3. Save the CMS page.
4. Open the CMS page in the frontend.

Expected behaviour: The iframe is displayed.
Actual behaviour: The iframe is not displayed at all.

### Questions or comments
See https://magento.stackexchange.com/a/312285/142.

### Contribution checklist (*)
